router/batchrouter/asyncdestinationmanager/common/utils.go

```diff
@@ -0,0 +1,20 @@
+package common
+
+import "slices"
+
+var (
+	asyncDestinations = []string{"MARKETO_BULK_UPLOAD", "BINGADS_AUDIENCE", "ELOQUA"}
```

Comment:
Should also have yandex destination in this list

Reply:
Yes, might have missed while resolving the conflicts. Added the yandex metric in async destinations list.